corehq/apps/hqadmin/tests/test_prbac.py

```diff
@@ -28,23 +28,10 @@ class TestCchqPrbacBootstrap(TestCase):
 
     @classmethod
     def setUpClass(cls):
+        super(TestCchqPrbacBootstrap, cls).setUpClass()
         Grant.objects.all().delete()
```

Comment:
Ah. ok so those are needed for some random unclean shutdowns elsewhere. thanks for looking into that